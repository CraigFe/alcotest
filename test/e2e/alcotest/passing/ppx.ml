module%test Lorem = struct
  let%test "ipsum" = ()
  let%test "dolor" = ()
end

module%test Sit = struct
  module%test Amet = struct
    let%test "consectetur" = ()
  end

  let%test "adipiscing" = ()
end

let () = print_endline ""

[%%run_tests]
