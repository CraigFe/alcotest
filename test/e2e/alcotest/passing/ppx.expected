
Testing `test/e2e/alcotest/passing/ppx.ml'.
This run has ID `<uuid>'.

  [PASS]    Lorem › ipsum
  [PASS]    Lorem › dolor
  [PASS]    Sit › Amet › consectetur
  [PASS]    Sit › adipiscing

Full test results in `<build-context>/_build/_tests/<test-dir>'.
Test Successful in <test-duration>s. 4 tests run.

